package.json

```diff
@@ -39,6 +39,9 @@
     ]
   },
   "jest": {
+    "moduleNameMapper": {
+      "\\.(css|scss)$": "<rootDir>/__mocks__/styleMock.js"
```

Comment:
We haven't had to do this so far, can we investigate why this is necessary and if there are any work arounds before this lands.